src/app/components/HeaderComponent.js

```diff
@@ -55,6 +55,7 @@ const HeaderComponent = (props) => {
             <LanguageSelectContainer />
             <StartButton />
           </Form>
+          <IndicatorLight />
```

Comment:
I think it looks better beside rif logo, at the left part.

Reply:
I can move it beside the logo. I think to the right of the logo (second screenshot) looks better though.

### left of the logo

![image](https://user-images.githubusercontent.com/766679/73929339-4755d080-48dd-11ea-8b9b-83d18cbacc40.png)

### right of the logo

![image](https://user-images.githubusercontent.com/766679/73929421-6f453400-48dd-11ea-907a-400b9984f192.png)

Reply:
Right of the logo 😁

Reply:
Moved with [807623e](https://github.com/rnsdomains/rns-manager-react/pull/123/commits/807623e78c693e9e4e5ef78c6ef4ad32454ee565)